spec.go

```diff
@@ -161,6 +163,13 @@ type ArtifactConfig struct {
 	Name string `yaml:"name,omitempty" json:"name,omitempty"`
 }
 
+// ArtifactConfigFileConfig is the configuration give for Artifact Config Files.
+// This is used to customize the settings for a config entry in the resulting artifact.
+type ArtifactConfigFileConfig struct {
+	ArtifactConfig
```

Comment:
```suggestion
	ArtifactConfig `yaml:",inline"`
```

Reply:
I've removed this entire struct now in favor of just Artifactconfig.